### DISCUSSION TITLE
Codeforces ödevi: Expression

### DISCUSSION THREAD

**Discussion by MASahin223** [sonsuzus/sonsuzus#462] [Soru Cevap] [answered]
Petya bir okulda okuyor ve Matematiği çok seviyor. Sınıfı aritmetik ifadeleri çalışıyor. Son derste öğretmen tahtaya üç pozitif tam sayı _a_, _b_, _c_ yazdı. Görev, ortaya çıkan ifadenin mümkün olduğunda büyük olması için sayılar arasına '+' ve '*' işlem işaretleri ve parantez eklemekti. Bir örneği ele alalım: Öğretmenin tahtaya 1, 2 ve 3 sayılarını yazdığını varsayalım. İşaret ve parantez yerleştirmenin bazı yolları şunlardır:

- 1 + 2 * 3=7
- 1 * (2 + 3)=5
- 1 * 2 * 3=6
- (1 + 2) * 3=9

Sadece _a_ ve _b_ arasına ve _b_ ve _c_ arasına işlem işaretleri ekleyebileceğinizi, yani tam sayıları değiştiremeyeceğinizi unutmayın. Örneğin, verilen örnekte (1 + 3) * 2 ifadesini elde edemezsiniz.

Elde edebileceğiniz maksimum değerin 9 olduğunu görürsünüz.

Göreviniz: _a_, _b_ ve _c_ verildiğinde elde edebileceğiniz maksimum değeri yazdırın.

**Girdi**
Giriş, her biri tek bir satırda olmak üzere _a_, _b_ ve _c_ üç tam sayısını içerir (1 ≤ _a_, _b_, _c_ ≤ 10).

**Çıktı**
Elde edebileceğiniz ifadenin maksimum değerini yazdırın.

### Örnekler
**Girdi**
```
1
2
3
```
**Çıktı**
```
9
```
---
**Girdi**
```
2
10
3
```
**Çıktı**
```
60
```

**Comment by MASahin223** [comment, marked as answer]
```python
a = int(input(""))
b = int(input(""))
c = int(input(""))
enbuyuk = 0
cevap = 0
buyuksecim = 0
while buyuksecim == 0:
    cevap = a+b+c
    if cevap > enbuyuk:
        enbuyuk = cevap
    cevap = a+b*c
    if cevap > enbuyuk:
        enbuyuk = cevap
    cevap = a*b+c
    if cevap > enbuyuk:
        enbuyuk = cevap
    cevap = (a+b)*c
    if cevap > enbuyuk:
        enbuyuk = cevap
    cevap = a*(b+c)
    if cevap > enbuyuk:
        enbuyuk = cevap
    cevap = a*b*c
    if cevap > enbuyuk:
        enbuyuk = cevap
    buyuksecim = 1
print("\n",enbuyuk,sep="")
```

**Reply by sonsuzus** [reply]
Aferin.
